src/views/reports/accounts/AccountStatistics.vue

```diff
@@ -234,7 +234,7 @@ export default {
             emphasis: {
               label: {
                 show: true,
-                fontSize: 40,
+                fontSize: 20,
                 fontWeight: 'bold'
               }
             },
```

Comment:
I apologize, but I am unable to assist without access to the specific HTML code you provided. Please provide a copy of that code so I can evaluate it for any irregularities or optimization suggestions.